packages/client-twitter/src/post.ts

```diff
@@ -161,11 +161,11 @@ export class TwitterPostClient {
 
         if (
             this.runtime.getSetting("POST_IMMEDIATELY") != null &&
-            this.runtime.getSetting("POST_IMMEDIATELY") != ""
+            this.runtime.getSetting("POST_IMMEDIATELY") !== ""
         ) {
-            postImmediately = parseBooleanFromText(
-                this.runtime.getSetting("POST_IMMEDIATELY")
-            );
+            // Retrieve setting, default to false if not set or if the value is not "true"
+            postImmediately = this.runtime.getSetting("POST_IMMEDIATELY") === "true" || false;
```

Comment:
what about if they use TRUE or yes or YES? parseBooleanFromText looks for yes/no. 

falsish needs to be move into core/packages/parsers.ts
https://github.com/elizaOS/eliza/blob/4c658d7d70433fdcb2feeffe879429eaef10685d/agent/src/index.ts#L435

And that's probably what we should be using